src/main/java/teammates/ui/controller/StudentProfilePictureDeleteAction.java

```diff
@@ -0,0 +1,27 @@
+package teammates.ui.controller;
+
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.util.Const;
+import teammates.common.util.StatusMessage;
+import teammates.common.util.StatusMessageColor;
+
+/**
+ * Action: deletes the profile picture.
+ */
+
+public class StudentProfilePictureDeleteAction extends Action {
```

Comment:
This class needs test at action level, e.g. `StudentProfilePictureDeleteActionTest`.